the-collab-lab/tcl-30-whats-near-me
# Add new API key

## Description

This PR adds a new API key and makes sure the app has access to Google Places API

## Acceptance Criteria

- App has access to Google Places API

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|     | :sparkles: New feature     |
| ✓ | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before

With the old API key, app had no access to Google Places API

### After

 Now, there's access to Google Places API

## Testing Steps / QA Criteria

1. In your terminal, from ``tcl30-whats-near-me``, pull down this branch ``ku-update-map-api-key``
2. Run ``npm start`` to launch the app
3. Make sure the map is displayed properly, and the app has access to Google Places API by searching for a location using searchbox in the map view
